presto-docs/src/main/sphinx/security/built-in-system-access-control.rst

```diff
@@ -77,13 +77,15 @@ contents:
 
 The config file is specified in JSON format.
 
-* It contains the rules defining which catalog can be accessed by which user (see Catalog Rules below).
+* The catalog rules specifying which catalog can be accessed by which user (see Catalog Rules below).
+* The schema rules specifying which users have ownership of which schemas (see Schema Rules below).
+* The table rules specifying which privileges are granted to which users over which tables (see Table Rules below).
 * The query rules specifying which queries can be managed by which user (see Query Rules below).
 * The impersonation rules specify which user impersonations are allowed (see Impersonation Rules below).
 * The principal rules specifying what principals can identify as what users (see Principal Rules below).
 * The system information rules specifying what users can access system management information (see System Information Rules below).
 
-This plugin currently supports catalog access, query, impersonation, principal, and system information
+This plugin currently supports catalog, schema, table access, query, impersonation, principal, and system information
```

Comment:
This whole sentence can be removed after the above list is done and contains all of these.

Reply:
Done.